File: json/php/read/json_read.php
#! /usr/bin/php
<?php
// ------------------------------------------------------------------
//	data_base/json/php/read/json_read.php
//
//					Sep/15/2016
//
// ------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
//
include_once "text_manipulate.php";
//
// ------------------------------------------------------------------
fputs (STDERR,"*** 開始 ***\n");
$json_filename = $argv[1];

$json_string = file_get_contents ($json_filename);
$dict_aa = json_decode ($json_string,true);

// var_dump ($root);

dict_display_proc ($dict_aa);

fputs (STDERR,"*** 終了 ***\n");

// ------------------------------------------------------------------
?>

File: json/dart/read/json_read.dart
/*
	json_read.dart

					Jan/09/2021
*/
// -------------------------------------------------------------
import 'dart:async';
import 'dart:io'; 
import 'dart:convert';

import	'/var/www/data_base/common/dart_common/text_manipulate.dart';
// -------------------------------------------------------------
main (List<String> arguments) {
	stderr.writeln ("*** 開始 ***");

	String file_name = '${arguments[0]}';

	File file = new File (file_name);
	Future future = file.readAsString ();

	future.then((content)
		{
		Map dict_aa = jsonDecode(content);

		dict_display_proc (dict_aa);

		stderr.writeln ("*** 終了 ***");
		}); 
}

// -------------------------------------------------------------
